.travis.yml

```diff
@@ -6,12 +6,12 @@ matrix:
     dist: trusty
     python: "2.7"
     env: LD_LIBRARY_PATH="$PWD/tmp-build/lib" LIBRDKAFKA_VERSION=master
-  # Source package verification with Python 2.7 and librdkafka v0.11.4
+  # Source package verification with Python 2.7 and librdkafka master
```

Comment:
If you rebase on master you can remove this diff since this section is no longer included.